apps/backoffice-v2/src/domains/business-reports/components/EcosystemAndTransactions/components/EcosystemAndTransactionsTable/columns.tsx

```diff
@@ -0,0 +1,81 @@
+import { TextWithNAFallback } from '@/common/components/atoms/TextWithNAFallback/TextWithNAFallback';
+import { checkIsUrl } from '@/common/utils/check-is-url/check-is-url';
+import { BallerineLink } from '@/common/components/atoms/BallerineLink/BallerineLink';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
+import { ctw } from '@/common/utils/ctw/ctw';
+import React from 'react';
+import { createColumnHelper } from '@tanstack/react-table';
+import { safeEvery } from '@ballerine/common';
+
+const columnHelper = createColumnHelper<{
+  matchedName: string;
+  relatedNodeType: string;
+  relatedNode: string;
+  indicators: {
+    label: string;
+    severity: string;
+  };
+}>();
+
+export const columns = [
+  columnHelper.display({
+    id: 'index',
+    cell: info => {
+      const index = info.cell.row.index + 1;
+
+      return <TextWithNAFallback className={`ps-8`}>{index}</TextWithNAFallback>;
+    },
+    header: 'Match',
+  }),
+  columnHelper.accessor('matchedName', {
+    header: 'Matched Name',
+    cell: info => {
+      const matchedName = info.getValue();
+      const matchedNameWithProtocol = `http://${matchedName}`;
```

Comment:
Please check that the `matchedName` is not a URL before adding the protocol to it